user-guide/Advanced_Modules/User Defined APIs/Tutorial/UD_APIs_Hello_world_tutorial.md

```diff
@@ -0,0 +1,142 @@
+                    ResponseCode = (int)StatusCode.Ok,
+                };
+            }
+        }
+    }
+    ```
+
+1. Now we will upload our automation script to DataMiner using the built-in feature in DIS. Make sure that DIS can connect to the DataMiner system you want to upload your script to, to add your DataMiner system, [edit the DIS settings](xref:DIS_settings#dma). In the Solution Explorer, open the HelloWorldAPI.xml file by double-clicking it. At the top you'll see a 'Publish' button. Click the arrow next to it and click the DataMiner system you want to upload your automation script to.  
+![Automation script XML](~/user-guide/images/UDAPIS_helloworld_11.jpg)  
+![Publish to DMA](~/user-guide/images/UDAPIS_helloworld_12.jpg)
+
+## Step 3: Create an API token
+
+To access our API, we require an [access token](xref:UD_APIs_Objects_ApiToken).
+
+1. Open DataMiner Cube, and login to your DataMiner system
+
+1. Open *System Center* > User-Defined APIs*. In this window we can manage our APIs and Tokens.
+
+1. Under *Tokens* click *Create...*
+
+1. Let's name our token 'HelloWorldToken', and click *Generate token*.
+![Create API Token](~/user-guide/images/UDAPIS_helloworld_13.jpg)
+
+1. A secret will now be generated, copy it, and **make sure you save it somewhere.** We will need it later to trigger our API, **you cannot retrieve the secret again after closing this window!** You can now see our token in the list of available tokens!
+![Create API secret](~/user-guide/images/UDAPIS_helloworld_14.jpg)
+
+![Tokens list](~/user-guide/images/UDAPIS_helloworld_15.jpg)
+
+## Step 4: Create the API Definition
+
+We will now create an API using the automation script and token we made earlier.
+
+1. Close the *System Center* in Cube and open the *Automation* module.
+
+1. You should see our *HelloWorldAPI* automation script in the left bar, click it to open it.
+
+1. At the bottom of the screen, there's a *Configure API...* button. Click it to start configuring our API.
+![Configure API... button](~/user-guide/images/UDAPIS_helloworld_16.jpg)
+
+1. Fill in a description, fill in 'helloworld' for the URL part, and select the 'HelloWorldToken' we made earler as access token, leave all other settings default. Then click *Create*.  
+![Create API window](~/user-guide/images/UDAPIS_helloworld_17.jpg)
+
+1. You should now see an icon as shown in the image next to your automation script, this shows that your API is live!
+![User-defined API icon on automation script](~/user-guide/images/UDAPIS_helloworld_18.jpg)
+
+## Optional step: Test the API using the SLNet Client Test Tool
+
+You can test your API using the SLNet Client Test Tool, how to do that is described on the [triggering and managing a user-defined API](xref:SLNetClientTest_triggering_api#triggering-a-user-defined-api) page. This approach is great for testing, but does not use an actual API call like we will use in the next step.
+
+## Step 5: Trigger the API using Postman
```

Comment:
I feel that this part needs a bit more screenshots. The UI for postman is not that clear. It could be hard to find the '+' in the first step. I would also add a screenshot with the URL filled in & one with the bearer token filled in.